third_party/move/move-vm/types/src/values/values_impl.rs

```diff
@@ -976,11 +1027,27 @@ impl IndexedRef {
 
 impl Value {
     pub fn equals(&self, other: &Self) -> PartialVMResult<bool> {
-        self.0.equals(&other.0)
+        // Note: no need to feature gate because values were previously bounded at pack time.
```

Comment:
Not sure whether a temporal comment like this is helpful in the future, but good to know for reviewers.

Reply:
Yep, it was mostly for review purpose, can remove if you prefer